pkg/reconciler/artifactbuild/controller.go

```diff
@@ -10,7 +10,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/source"
 
 	"github.com/redhat-appstudio/jvm-build-service/pkg/apis/jvmbuildservice/v1alpha1"
-	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	v1beta1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1"
```

Comment:
Question - why is this named with beta when its v1 (i.e. couldn't it be e.g. `pipelinev1`)  ?

Reply:
To stop a massive amount of conflicts with your PR (and every other PR).

The envtest failure is real and I am not sure how to fix it. I don't think this test really adds much value, so to get this in we might just have to delete it.

Reply:
Ah I see, yes that would be painful :-) Is the plan to change them in future during a 'quiet' period?

Reply:
yea, that was the plan.